configs/src/deployments/json/modeMainnet.json

```diff
@@ -0,0 +1,184 @@
+    },
+    "PublicResolver": {
+      "address": "0xc4ddBd0472C2688F5d278dAFC02E66EE176C8DE3",
+      "blockNumber": 13626902,
+      "deploymentTx": "0xfe5a86cb85f9c0c4dd53fe5dec2a4c67ec0bf29d447f9e2d81e86413cd73b295"
+    },
+    "TokenVotingSetup": {
+      "address": "0x96E54098317631641703404C06A5afAD89da7373",
+      "blockNumber": 13627023,
+      "deploymentTx": "0x6e961c3c4558e8bab2f74e3ffe60c56afed1a6ef4fa2a85db3ed6cc3d8344fc8"
+    },
+    "TokenVotingSetupImplementation": {
+      "address": "0x4f40F01b2944359A0E507b509a868C8DbFAd215B",
+      "blockNumber": 13627023,
+      "deploymentTx": "0x6e961c3c4558e8bab2f74e3ffe60c56afed1a6ef4fa2a85db3ed6cc3d8344fc8"
+    },
+    "AddresslistVotingRepoProxy": {
+      "address": "0xf415FF95166EF5D365fFB3bc6d1701f9e9ed7Df7",
+      "blockNumber": 13627039,
+      "deploymentTx": "0x859f481098803fbc43745566b8b32e7568cf53fac310c54fda555a3310b6af4e"
+    },
+    "AddresslistVotingRepoImplementation": {
+      "address": "0x7a62da7B56fB3bfCdF70E900787010Bc4c9Ca42e",
+      "blockNumber": 13627009,
+      "deploymentTx": "0xc3e18ea335f8f00f6b192c228ca3ac9508b5837636a20e1f491f3f5b35d19494"
+    },
+    "AdminRepoProxy": {
+      "address": "0x326A2aee6A8eE78D79E7E956DE60C6E452f76a8e",
+      "blockNumber": 13627066,
+      "deploymentTx": "0x3a633c367c2f4ed2911570377d37a5631bb7e6fad058e0f55140d3210f108c24"
+    },
+    "AdminRepoImplementation": {
+      "address": "0xE978942c691e43f65c1B7c7F8f1dc8cDF061B13f",
+      "blockNumber": 13627066,
+      "deploymentTx": "0x3a633c367c2f4ed2911570377d37a5631bb7e6fad058e0f55140d3210f108c24"
+    },
+    "ManagementDAOProxy": {
+      "address": "0xF3AaA3372EbBf01b923a4Cc98Cd847126b3D73cA",
+      "blockNumber": 13626882,
+      "deploymentTx": "0xa25407d3c0fd9db57545a65f70f25eefd60ee1fcc161038539d67662f8dcf53f"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0x741061B01d3A95a7734B594f1884b6c30E0fDA96",
+      "blockNumber": 13626879,
+      "deploymentTx": "0x794ba6998e7ac7cee6c7fa505edceaf975c4191541649cf364fd921b55eaa3de"
+    },
+    "MultisigRepoProxy": {
+      "address": "0x7553E6Fb020c5740768cF289e603770AA09b7aE2",
+      "blockNumber": 13627076,
+      "deploymentTx": "0x446005fbd528deaf5b2ad35b146f13ffa7cacaa96abbe9292b6fa19451a939f7"
+    },
+    "MultisigRepoImplementation": {
+      "address": "0xA03C2182af8eC460D498108C92E8638a580b94d4",
+      "blockNumber": 13627031,
+      "deploymentTx": "0x7058071c479c267c687f6f3a80710003c28f867abe2ec819b813bc107e8ee492"
+    },
+    "TokenVotingRepoProxy": {
+      "address": "0x1AeD2BEb470aeFD65B43f905Bd5371b1E4749d18",
+      "blockNumber": 13627053,
+      "deploymentTx": "0xca53ea495e6b58ee198c9ebc26cad6ab61d6f120ca90cdad91feba4d3c0fe5ca"
+    },
+    "TokenVotingRepoImplementation": {
+      "address": "0x96E54098317631641703404C06A5afAD89da7373",
+      "blockNumber": 13627023,
+      "deploymentTx": "0x6e961c3c4558e8bab2f74e3ffe60c56afed1a6ef4fa2a85db3ed6cc3d8344fc8"
+    }
```

Comment:
this should be the same as pluginRepoBase not tokenVotingSetup, right?

Reply:
Unfortunately not.
The name is very confusing, but this is supposed to point to the plugin implementation, rather than the repo implementation.

I am keeping things like this only due to consistency reasons

Reply:
Please check mainnet, this implementation points to proxy impl.
if we go what u r suggesting ,, then we have to modify all other jsons

the objective of this is to determine which repo proxy got updated or not

Reply:
- Why publishing such address when `PluginRepoBase` is already listed on this file?
- I am positive that on other networks `TokenVotingRepoImplementation` was pointing to the plugin implementation (which feels wrong). If `TokenVotingRepoImplementation` points to PluginRepoBase on mainnet, then there should be a separate PR ensuring that all keys point to the exact same type of contract (out of the scope of this one)
- Glad to make it point to PluginRepoBase here (redundant), but then we don't export the plugin address